decidim/decidim
# Add Kebab menu for resources

#### :tophat: What? Why?
This PR partially covers the kebab related changes from #13328. In this PR we handle only the follow and report buttons that needs to be hidden. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13328

#### Testing
1. Make sure the pipeline is green
2. Apply patch, and see that all the follow and report buttons are within kebab

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
Meeting: 
![image](https://github.com/user-attachments/assets/6090433f-8572-4974-af13-bc68bd4cf9a6)
![image](https://github.com/user-attachments/assets/b4663a35-706d-4c46-8f39-ac3173b1b67f)

Debates 
![image](https://github.com/user-attachments/assets/4268a421-303d-4da7-9cc3-dfaf75910901)
![image](https://github.com/user-attachments/assets/2532b75e-e9d9-4eb2-9e7f-e3795332d420)

Proposals:
![image](https://github.com/user-attachments/assets/a10851bf-6265-4dd3-829d-81b155eae218)
![image](https://github.com/user-attachments/assets/88e1acb9-3ca0-44df-af70-c3cd80382bc3)

Post
![image](https://github.com/user-attachments/assets/0433d520-a373-49d5-af55-88797c6458f1)
![image](https://github.com/user-attachments/assets/9a8046ed-430f-46ce-b433-ba432ff95ff2)

Budget:
![image](https://github.com/user-attachments/assets/9b601bee-8784-4d0b-999d-973e49a753a6)
![image](https://github.com/user-attachments/assets/a4c814d8-2c2b-4e78-90f8-b1c584acedf6)


:hearts: Thank you!
